.gitmodules

```diff
@@ -1,3 +1,3 @@
-[submodule "test-harness"]
-	path = test-harness
-	url = https://github.com/open-feature/test-harness
+[submodule "spec"]
```

Comment:
i think it's not required to call this spec. Alternatively we could keep calling i "test-harness" (same for path).
no hard feeling there tho. wdyt?

Reply:
i would go with spec, to be more precise, as we are using spec and test-harness in the contrib repos